vcd/resource_vcd_snat_test.go

```diff
@@ -63,6 +67,20 @@ func TestAccVcdSNAT_Basic(t *testing.T) {
 						"vcd_snat."+snatName, "internal_ip", startIpAddress),
 				),
 			},
+			resource.TestStep{
+				Config: updateText,
+				// TODO remove with Terraform 0.12 support
```

Comment:
We are at Terraform 0.12. Maybe we can remove these comments?

Reply:
removed comments